app/assets/javascripts/modules/orgs.js

```diff
@@ -13,6 +13,22 @@ angular.module('probedock.orgs', [ 'probedock.api', 'probedock.auth', 'probedock
         service.emit('refresh', service.organizations);
       },
 
+      getOrganization: function(orgId) {
+        var org =_.findWhere(service.organizations, { id: orgId });
+
+        if (_.isUndefined(org)) {
+          return api({
+            url: '/organizations/' + orgId
+          })
+          .then(function(response) {
```

Comment:
I Fixed it in your way but be prepared to do the remark again an undefined number of times as I will continue to use my convention in other projects :)